scripts/flashing/dlink_recovery_upload.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: MIT
+#
+#   D-Link COVR-P2500 flash utility
+#
+#   Upload firmware to the device with waiting for ACK after each TCP packet.
```

Comment:
Attaching this file to the wiki should be more useful than integrating it into the source file.

Reply:
There are lots of scripts in that folder, e.g. for AVM FritzBoxes. I think it's quite handy to have them checked out automatically with OpenWrt, but for the average user they should of course be mentioned in the wiki as well.

Reply:
Attached script to [wiki](https://openwrt.org/inbox/toh/d-link/covr-p2500_a1#dlink_recovery_uploadpy) and removed from git